packages/flame/lib/src/components/position_component.dart

```diff
@@ -230,4 +361,4 @@ abstract class PositionComponent extends BaseComponent {
     size.setValues(rect.width, rect.height);
     topLeftPosition = rect.topLeft.toVector2();
   }
-}
+} // ignore: number-of-methods
```

Comment:
I think we can just bump the codemetrics baseline instead, I don't think it's necessarily a good metric anyway and I prefer to avoid "ignoring" rules

Reply:
I've upped the `number-of-methods` parameter to 50